.gitignore

```diff
@@ -21,6 +24,14 @@ DDT_DATA/testReports*/*
 
 TEMP_DATA
 
+# Generator and run config files. Ignore non-standard ones
+gen*.sh
+run*.json*
+run*.sh
+!generateDataAndRun.sh
+!run_config.json
+!genData100.sh    
```

Comment:
This seems heavy handed to ignore patterns of files yet allow-list specific exceptions for very important files. The way `.gitignore` files should be used is to list file patterns or specific files that we don't ever want to consider important.

If you have files that fit any of the patterns above that you are trying to ignore, then they are either:
1. temporary files that you should get rid of once they've served their purpose
2. important files that should be checked in

Reply:
Hmm, I have a set of gen scripts that I used for quick tests, e.g., testing only ICU75. But I don't think they are relevant to everyone.

Reply:
Let's remove the remaining 3 lines (which are now the new lines 28-30) that are exceptions to the previous 3 lines (the previous lines 28-30) since the exceptions are now superfluous.